easy-ui-react/src/Button/Button.module.scss

```diff
@@ -0,0 +1,285 @@
+      var(--ezui-shadow-button)
+    );
+  }
+
+  &.colorWarningFilled {
+    @include filled-button(
+      var(--ezui-color-blue-800),
+      var(--ezui-color-blue-800),
+      var(--ezui-color-red-500),
+      var(--ezui-color-red-600),
+      var(--ezui-color-red-700),
+      var(--ezui-shadow-button)
+    );
+  }
+
+  &.colorNeutralFilled {
+    @include filled-button(
+      var(--ezui-color-white),
+      var(--ezui-color-white),
+      var(--ezui-color-blue-800),
+      var(--ezui-color-black),
+      var(--ezui-color-blue-700),
+      var(--ezui-shadow-button)
+    );
+  }
+
+  &.colorPrimaryOutlined {
+    @include outlined-button(
+      var(--ezui-color-blue-500),
+      var(--ezui-color-blue-600),
+      var(--ezui-color-blue-700),
+      var(--ezui-shadow-button)
+    );
+  }
+
+  &.colorSupportOutlined {
+    @include outlined-button(
+      var(--ezui-color-blue-800),
+      var(--ezui-color-black),
+      var(--ezui-color-gray-700),
+      var(--ezui-shadow-button)
+    );
+  }
+
+  &.colorSuccessOutlined {
+    @include outlined-button(
+      var(--ezui-color-green-500),
+      var(--ezui-color-green-600),
+      var(--ezui-color-green-700),
+      var(--ezui-shadow-button)
+    );
+  }
+
+  &.colorSecondaryOutlined {
+    @include outlined-button(
+      var(--ezui-color-purple-500),
+      var(--ezui-color-purple-600),
+      var(--ezui-color-purple-700),
+      var(--ezui-shadow-button)
+    );
+  }
+
+  &.colorWarningOutlined {
+    @include outlined-button(
+      var(--ezui-color-red-500),
+      var(--ezui-color-red-600),
+      var(--ezui-color-red-700),
+      var(--ezui-shadow-button)
+    );
+  }
+
+  &.colorInverseOutlined {
+    @include outlined-button(
+      var(--ezui-color-white),
+      var(--ezui-color-gray-100),
+      var(--ezui-color-gray-200),
+      var(--ezui-shadow-button)
+    );
+  }
+
+  &.colorPrimaryLink {
+    @include link-button(
+      var(--ezui-color-blue-500),
+      var(--ezui-color-blue-600),
+      var(--ezui-color-blue-700),
+      var(--ezui-shadow-button)
+    );
+  }
+
+  &.colorSecondaryLink {
+    @include link-button(
+      var(--ezui-color-purple-500),
+      var(--ezui-color-purple-600),
+      var(--ezui-color-purple-700),
+      var(--ezui-shadow-button)
+    );
+  }
+
+  &.sizeMdFilled,
+  &.sizeMdOutlined,
+  &.sizeMdLink {
+    @include font-style("button");
+  }
+
+  &.sizeSmFilled,
+  &.sizeSmOutlined,
+  &.sizeSmLink {
+    @include font-style("small-button");
+  }
+
+  &.sizeMdFilled {
+    padding: var(--ezui-space-1-5) var(--ezui-space-2);
+  }
+
+  &.sizeMdOutlined,
+  &.sizeMdLink {
+    padding: calc(
+        #{var(--ezui-space-1-5)} - #{var(--ezui-shape-border-width-1)}
+      )
+      calc(#{var(--ezui-space-2)} - #{var(--ezui-shape-border-width-1)});
+  }
+
+  &.sizeSmFilled {
+    padding: var(--ezui-space-1);
+  }
+
+  &.sizeSmOutlined,
+  &.sizeSmLink {
+    padding: calc(#{var(--ezui-space-1)} - #{var(--ezui-shape-border-width-1)});
+  }
```

Comment:
will double check with design about the height for the small button, i presume its intended to be 32px, but the figma has 36px due to the previous line height